torch/csrc/api/src/nn/modules/transformer.cpp

```diff
@@ -186,9 +187,8 @@ Tensor TransformerDecoderLayerImpl::activation(const Tensor& input){
   } else if (c10::get_if<enumtype::kReLU>(&options.activation())) {
     return F::relu(input);
   } else {
-    TORCH_CHECK(false,
-      "Unknown activation: ",
-      torch::enumtype::get_enum_name(options.activation()));
+    auto callable_activation = *c10::get_if<std::function<Tensor(const Tensor&)> >(&options.activation());
```

Comment:
Same question as above

Reply:
Adding unconditional else back in as a separate branch